backend/handlers/login.js

```diff
@@ -1,60 +0,0 @@
-const user = require('../../database/userhelpers.js');
-const Boom = require('boom');
-const Bcrypt = require('bcrypt');
-module.exports = (req, res) => {
-  const email = req.payload.email;
-  const password = req.payload.password;
-  user.getuserbyemail(email, (error, result) => {
```

Comment:
in general, consider using camelCase for your function and variable names. Makes it more readable and easier to reuse.